tests/config/config.d/heavy_metrics.xml

```diff
@@ -0,0 +1,5 @@
+<clickhouse>
```

Comment:
Heavy metrics are always updated on `SYSTEM RELOAD ASYNCHRONOUS METRICS`, so let's rely on this instead of these settings.

Reply:
IIRC, they are disabled in tests by default. But in any case, we shouldn't enable them in all the tests. I think it's better to write a small integration test instead

Reply:
Looking at the code ([one](https://github.com/ClickHouse/ClickHouse/blob/4c7876f11324fbc185aa562aa970c9f23ddef619/src/Interpreters/InterpreterSystemQuery.cpp#L656), [two](https://github.com/ClickHouse/ClickHouse/blob/4c7876f11324fbc185aa562aa970c9f23ddef619/src/Interpreters/ServerAsynchronousMetrics.cpp#L504)), they should be reloaded on `SYSTEM RELOAD ASYNCHRONOUS METRICS` anyway because `force_update` is always `true`.

Reply:
So, no extra settings should be needed, and a functional test should suffice

Reply:
Not really, `updateHeavyMetricsIfNeeded` is called [here](https://github.com/ClickHouse/ClickHouse/blob/4c7876f11324fbc185aa562aa970c9f23ddef619/src/Interpreters/ServerAsynchronousMetrics.cpp#L431C9-L432), and only when `update_heavy_metrics` is true. And it's only true, when the setting is enabled: https://github.com/ClickHouse/ClickHouse/blob/4c7876f11324fbc185aa562aa970c9f23ddef619/programs/server/Server.cpp#L1198-L1201